File: 2529-maximum-count-of-positive-integer-and-negative-integer/2529-maximum-count-of-positive-integer-and-negative-integer.java
class Solution {
    public int maximumCount(int[] nums) {
        int negCount = 0;
        int posCount = 0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]>0){
                posCount++;
            }
            if(nums[i]<0){
                negCount++;
            }
        }
        return Math.max(posCount,negCount);
    }
}

File: 2529-maximum-count-of-positive-integer-and-negative-integer/2529-maximum-count-of-positive-integer-and-negative-integer.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var maximumCount = function(nums) {
    let neg = 0, pos = 0;
    for(let i=0;i<nums.length;i++){
        if(nums[i] > 0){
            pos++;
        }else if(nums[i] < 0){
            neg++;
        }
    }
    return Math.max(pos,neg);
};